source/geh_common/src/geh_common/data_products/measurements_calculated/calculated_measurements_v1.py

```diff
@@ -22,6 +22,12 @@
         # A datetime value indicating when the transaction was created by the calculation job.
         t.StructField("transaction_creation_datetime", t.TimestampType(), not nullable),
         #
+        # A datetime value indicating the first observation time.
+        t.StructField("transaction_start_time", t.TimestampType(), not nullable),
+        #
+        # A datetime value indicating the last observation time (plus the resolution).
```

Comment:
The comment is implementation detail.
```suggestion
        # A datetime value indicating the end of the transaction (which equals the last observation time plus the resolution).
```